README.md

```diff
@@ -363,10 +364,28 @@ If you need a higher limit, drop us a line at support@scalyr.com.
 
 ## Fetching numeric data using a timeseries
 
-The "timeseries-query" command allows you to retrieve numeric data using a timeseries defined using the
-create-timeseries command. (Note that the [Scalyr API](https://www.scalyr.com/help/api#timeseriesQuery)
+A timeseries precomputes a numeric query, allowing you to execute queries almost instantaneously, and without
+exhausting your query execution limit.  This is especially useful if you are using the Scalyr API to feed a
+home-built dashboard, alerting system, or other automated tool. Note that the [Scalyr API](https://www.scalyr.com/help/api#timeseriesQuery)
 allows multiple timeseries queries in a single API invocation, but the command-line tool only supports
-one query at a time.)
+one query at a time.
+
+When a new timeseries is defined, we immediately start live updating of that timeseries from the ingestion pipeline.
+In addition, we begin a background process to extend the timeseries backward in time, so that it covers the full
+timespan of your query. This backfill process is automatic, and if you later issue the same query with an even
+earlier start time, we will extend the backfill to cover that as well.
+To change this behavior, use `--no-create-summaries`.
+
+A related argument, `--only-use-summaries`, controls whether this API call should only use preexisting timeseries or should
+actually execute the queries against the event database. If the argument is used, then your API call is guaranteed to return
```

Comment:
`s/actually //`, `s/against the event database/\0 if no matching summary exists/`, `s/If the argument/If this argument/`